content/applications/hr/employees/new_employee.rst

```diff
@@ -57,22 +58,22 @@ Optional fields
         :align: center
         :alt: Both job position fields entered but with different information.
 
-- Tags: Click on a tag in the drop-down menu to add any tags applicable to the employee. Any tag can
+- Tags: click on a tag in the drop-down menu to add any tags applicable to the employee. Any tag can
   be created in this field by typing it in. Once created, the new tag is available for all employee
   records. There is no limit to the amount of tags that can be added.
 - Work Contact Information: Enter the employee's :guilabel:`Work Mobile`, :guilabel:`Work Phone`,
```

Comment:
Weird- I see the word "click" as lower-case in my doc already, so not sure why it doesn't appear that way.

As for the work info, it should not because that's just a phrase used to describe the section where the work info is housed. That phrase does not appear anywhere on the form, and is a way for me to describe the left side of the card, in the top section.